src/filterFuntion/ratingFilter.js

```diff
@@ -0,0 +1,7 @@
+export const ratingFilter = (product, rating) => {
+    if (rating === "fiveStar")
+      return product.filter((a) => a.rating === 5);
+    if (rating === "fourStar")
+      return product.filter((a) => a.rating === 4);
+    return product;
+  };
```

Comment:
You see to be missing `twoStar` and `threeStar` rating filters?

Reply:
ok i will add it